crates/pathfinder/src/rpc/types.rs

```diff
@@ -693,10 +693,9 @@ pub mod reply {
                                 common: CommonTransactionProperties {
                                     hash: txn.transaction_hash,
                                     max_fee: txn.max_fee,
-                                    // no `version` in invoke transactions
-                                    version: TransactionVersion(Default::default()),
+                                    version: TransactionVersion(web3::types::H256::zero()),
```

Comment:
Wasn't there a const for this? Can't remember now :D

Could be nice to add if not.

Reply:
No const that I'm aware of -- but at least zero() is a const fn: https://docs.rs/web3/0.18.0/web3/types/struct.H256.html#method.zero

Reply:
Ah right I missed that its not our type somehow.

Reply:
Actually, should we not make it our inner type? (not a problem with this PR though).

Reply:
We could I think -- the reason we chose H256 was that the limits used by the `cairo-lang` code were too large for us to represent the version in an `u128`: 

```python
TRANSACTION_VERSION_LOWER_BOUND = 0
TRANSACTION_VERSION_UPPER_BOUND = FIELD_SIZE
```

Reply:
Hmm. I'll ask if that's an actually reasonable limit.